backend/module/chaos/serverexperimentation/rtds/rtds_cache_test.go

```diff
@@ -8,74 +8,92 @@ import (
 	gcpTypes "github.com/envoyproxy/go-control-plane/pkg/cache/types"
 	gcpCache "github.com/envoyproxy/go-control-plane/pkg/cache/v2"
 	gcpResource "github.com/envoyproxy/go-control-plane/pkg/resource/v2"
+	"github.com/golang/protobuf/ptypes"
 	pstruct "github.com/golang/protobuf/ptypes/struct"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/zap"
 
 	experimentation "github.com/lyft/clutch/backend/api/chaos/experimentation/v1"
+	serverexperimentation "github.com/lyft/clutch/backend/api/chaos/serverexperimentation/v1"
 )
 
-func createAbortExperiment(upstreamCluster string, downstreamCluster string, faultPercent float32, httpStatus int32) *experimentation.Experiment {
-	return &experimentation.Experiment{
-		TestSpecification: &experimentation.TestSpecification{
-			Config: &experimentation.TestSpecification_Abort{
-				Abort: &experimentation.AbortFault{
-					Target: &experimentation.AbortFault_ClusterPair{
-						ClusterPair: &experimentation.ClusterPairTarget{
-							DownstreamCluster: downstreamCluster,
-							UpstreamCluster:   upstreamCluster,
-						},
+func createAbortExperiment(t *testing.T, upstreamCluster string, downstreamCluster string, faultPercent float32, httpStatus int32) *experimentation.Experiment {
+	testConfig := &serverexperimentation.TestSpecification{
+		Config: &serverexperimentation.TestSpecification_Abort{
+			Abort: &serverexperimentation.AbortFault{
+				Target: &serverexperimentation.AbortFault_ClusterPair{
+					ClusterPair: &serverexperimentation.ClusterPairTarget{
+						DownstreamCluster: downstreamCluster,
+						UpstreamCluster:   upstreamCluster,
 					},
-					Percent:    faultPercent,
-					HttpStatus: httpStatus},
-			},
+				},
+				Percent:    faultPercent,
+				HttpStatus: httpStatus},
 		},
 	}
+
+	anyConfig, err := ptypes.MarshalAny(testConfig)
+	if err != nil {
+		t.Errorf("setSnapshot failed %v", err)
```

Comment:
This looks like copy & pasta =)

```suggestion
		t.Errorf("couldn't marshal experiment config: %v", err)
```